common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -134,6 +135,29 @@ public TachyonConf() {
       throw new RuntimeException("Unable to load default Tachyon properties file.", e);
     }
 
+    // Override default properties if there exist corresponding environment variables
+    if (getenv("TACHYON_MASTER_ADDRESS") != null) {
+      defaultProps.setProperty(Constants.MASTER_HOSTNAME, getenv("TACHYON_MASTER_ADDRESS"));
+    }
+    if (getenv("TACHYON_MASTER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_BIND_HOST, getenv("TACHYON_MASTER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_MASTER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_WEB_BIND_HOST,
+          getenv("TACHYON_MASTER_WEB_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_BIND_HOST, getenv("TACHYON_WORKER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_DATA_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_DATA_BIND_HOST,
+          getenv("TACHYON_WORKER_DATA_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_WEB_BIND_HOST,
+          getenv("TACHYON_WORKER_WEB_BIND_HOST"));
+    }
+
```

Comment:
In current design, all binding environment variables have to be loaded into defaultProps, siteProps or systemProps, which can avoid later direct retrievement on the environment variables from individual call. So should I load these variables into systemProps instead of defaultProps?